packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -48,38 +48,33 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
 
   public async request<T>(args: RequestArguments): Promise<T> {
     try {
-      const invalidArgsError = checkErrorForInvalidRequestArgs(args);
-      if (invalidArgsError) throw invalidArgsError;
+      checkErrorForInvalidRequestArgs(args);
```

Comment:
this function returns an error right, it doesn't throw the error. So this doesn't do anything since the throw was removed?

Reply:
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/be8fb5d4-d3c7-4577-8dc8-48739a992f5e)
@nateReiners